### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by mzeeshan3597**
docs added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# test-git
```
Comment (mzeeshan7100) [inline README.md:1]: change it

Comment (mzeeshan3597) [reply README.md:1]: nop

**Review by mzeeshan7100 — CHANGES_REQUESTED** [review]
change titile

**Review by mzeeshan7100 — COMMENTED** [review]
...,.kl;,m';j

**Review by mzeeshan7100 — APPROVED** [review]

**Pull request merged**